File: BEmuJava/bemu/src/main/java/com/bloomberglp/blpapi/MarketMessageSessionOpened.java
//------------------------------------------------------------------------------
// <copyright project="BEmu_maven" file="/BEmu_maven/bemu/src/main/java/com/bloomberglp/blpapi/MarketMessageSessionOpened.java" company="Jordan Robinson">
//     Copyright (c) 2013 Jordan Robinson. All rights reserved.
//
//     The use of this software is governed by the Microsoft Public License
//     which is included with this distribution.
// </copyright>
//------------------------------------------------------------------------------

package com.bloomberglp.blpapi;

import com.bloomberglp.blpapi.Name;
import com.bloomberglp.blpapi.Message;

public class MarketMessageSessionOpened extends Message
{
    MarketMessageSessionOpened() throws Exception
    {
    	super(new Name("SessionStarted"), null, null);
    }

    public String topicName()
    {
    	return "";
    }
}

File: BEmu/MarketDataRequest/MarketMessageSessionOpened.cs
//------------------------------------------------------------------------------
// <copyright project="BEmu_csh" file="MarketDataRequest/MarketMessageSessionOpened.cs" company="Jordan Robinson">
//     Copyright (c) 2013 Jordan Robinson. All rights reserved.
//
//     The use of this software is governed by the Microsoft Public License
//     which is included with this distribution.
// </copyright>
//------------------------------------------------------------------------------

namespace Bloomberglp.Blpapi.MarketDataRequest
{
    using System;
    using System.Collections.Generic;
    using System.Linq;
    using System.Text;

    internal class MarketMessageSessionOpened : Message
    {
        internal MarketMessageSessionOpened() : base(new Name("SessionStarted"), null, null)
        {
        }

        public override string TopicName { get { return ""; } }

        public override IEnumerable<Element> Elements
        {
            get { throw new NotImplementedException(); }
        }
    }
}
